user-registration-app/src/app/views/registration-form/registration-form.component.html

```diff
@@ -192,7 +192,7 @@ <h5>Are you a veteran?</h5>
   <div class="row scrollspy" materialize="scrollSpy" id="age-container">
     <div class="card medium darken-1">
       <div class="card-content ">
-        <h5 class="age">Will you be at least 18  or a current university student before November 2, 2019?</h5>
+        <h5 class="age">Will you be at least 18 or a current Penn State University student before April 4, 2020?</h5>
```

Comment:
Are we only allowing PSU students this semester?

Reply:
If they're minors yeah